### PR TITLE
Displaying a message if user does not own any stocks and is on sell stocks page.

### PR DESCRIPTION
**Pull request by Devashish0210**
fixed an issue where even if the user had 0 stocks of a certain company, the stock card of that company would still show with quantity 0. so instead of that, a message saying "You don't have any stocks yet. Click below to buy some" has been added.
Issue : Display message if user does not hold any stock or have sold all of them. #206

### CODE REVIEW

File: pages/trading/sell.js [hunk 1]
```diff
@@ -34,6 +34,12 @@ const SellStocks = () => {
     );
   };
 
+  const availableStocks = userStocks
+    .filter((itemName) => itemName.quantity)
+    .map((itemName) => (
+      <Card key={itemName.id} stock={itemName} operationType={'SELL'} />
+    ));
+  const Message = "You don't have any stocks yet. Click below to buy some";
```
[the remark this answers is not in the document]
Comment (swarajpure) [reply pages/trading/sell.js:42]: Also, this better to be declared at the top of the file

**Review by swarajpure — CHANGES_REQUESTED** [review]
Requesting a change in variable name.
Please check.

Good job! ⚡

**Comment by Devashish0210** [comment]
Initial
As we can see in the initial image, even if the user does not own any stocks of the company, the company card along with "Quantity : 0" is still unwantedly mentioned there. That has been changed, the cards have been removed if the user does not own any stocks and a simple message has been inserted telling the user to buy more stocks. See image below for reference.
Final Render

**Comment by Devashish0210** [comment]
Initial
As we can see in the initial image, even if the user does not own any stocks of the company, the company card along with "Quantity : 0" is still unwantedly mentioned there. That has been changed, the cards have been removed if the user does not own any stocks and a simple message has been inserted telling the user to buy more stocks. See image below for reference.
Final Render

**Comment by Devashish0210** [comment]
Initial
As we can see in the initial image, even if the user does not own any stocks of the company, the company card along with "Quantity : 0" is still unwantedly mentioned there. That has been changed, the cards have been removed if the user does not own any stocks and a simple message has been inserted telling the user to buy more stocks. See image below for reference.
Final Render

**Comment by Devashish0210** [comment]
Initial
As we can see in the initial image, even if the user does not own any stocks of the company, the company card along with "Quantity : 0" is still unwantedly mentioned there. That has been changed, the cards have been removed if the user does not own any stocks and a simple message has been inserted telling the user to buy more stocks. See image below for reference.
Final Render

**Review by swarajpure — APPROVED** [review]
Looks good to me.

Thanks!

**Pull request merged**